pages/index.tsx

```diff
@@ -120,6 +120,11 @@ export default function Home({
               icon: <InsertPhoto />,
               href: "/category/digitalimageprocessing",
             },
+            {
+              name: t("categories:strings"),
+              icon: <TextSnippet />,
```

Comment:
Seems this is an invalid icon. Not sure where can we see the list of icons, though. @maltejur?

Reply:
The icons can be found here https://mui.com/material-ui/material-icons/